staging/src/k8s.io/apiserver/pkg/apis/config/validation/validation_test.go

```diff
@@ -415,11 +451,24 @@ func TestKMSProviderName(t *testing.T) {
 				field.Invalid(nameField, "foo:bar", fmt.Sprintf(invalidKMSConfigNameErrFmt, "foo:bar")),
 			},
 		},
+		{
+			desc: "invalid name with : but api version is v1",
```

Comment:
Need more tests around colon right?

Reply:
We already have tests for `:` with v2. The test for v1 was missing, so I added that. Let me know if we should add more.